src/Symfony/Component/ClassLoader/Tests/ApcClassLoaderTest.php

```diff
@@ -19,7 +19,7 @@ class ApcClassLoaderTest extends TestCase
 {
     protected function setUp()
     {
-        if (!(ini_get('apc.enabled') && ini_get('apc.enable_cli'))) {
+        if (!(\filter_var(ini_get('apc.enabled'), \FILTER_VALIDATE_BOOLEAN) && \filter_var(ini_get('apc.enable_cli'), \FILTER_VALIDATE_BOOLEAN))) {
```

Comment:
Can you please remove all the `\` added in front of `filter_var()` and `FILTER_VALIDATE_BOOLEAN` in the PR?
We don't add these except for a specific short list of functions.

Reply:
Ok, changed ;)